README.md

```diff
@@ -1,21 +1,24 @@
 # Capstone_Appointment_back_end
 
-> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface. This app uses RoR, RSPec, Capybara, Devise, and CanCanCan.
+> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface.
```

Comment:
- [ ] A screenshot would be wonderful for this readme doc.  It could be your UML or even this (your swagger docs!):

![image](https://user-images.githubusercontent.com/57421823/199776952-c0589eec-f902-4125-95ff-56c5a76d645c.png)

Please, kindly add a screenshot to further illustrate your API readme.  It will look awesome, and professional 💪